packages/dendron-cli/src/commands/visualizeCLICommand.ts

```diff
@@ -0,0 +1,34 @@
+import { generateSVG, InputArgs } from "@dendronhq/dendron-viz";
+import { Argv } from "yargs";
+import { CLICommand, CommandCommonProps } from "./base";
+import { setupEngine, setupEngineArgs, SetupEngineResp } from "./utils";
+
+type CommandOpts = InputArgs & SetupEngineResp & CommandCommonProps;
+
+export { CommandOpts as VisualizeCLICommandOpts };
+
+export class VisualizeCLICommand extends CLICommand {
+  constructor() {
+    super({
+      name: "visualize",
+      desc: "generates a fingerprint visualization of Dendron workspace",
```

Comment:
It looks like "packed circles" is the right term for this type of visualization, can we use that name for the description: https://datavizproject.com/data-type/packed-circle-chart/